use-cases/school-timetabling/src/main/resources/application.properties

```diff
@@ -29,7 +29,7 @@ quarkus.log.category."ai.timefold.solver".level=DEBUG
 %prod.quarkus.log.category."ai.timefold.solver".level=INFO
 
 # XML file for power tweaking, defaults to solverConfig.xml (directly under src/main/resources)
-# quarkus.timefold.solver-config-xml=org/.../timeTableSolverConfig.xml
+# quarkus.timefold.solver-config-xml=org/.../timetableSolverConfig.xml
```

Comment:
I see property change, but no file name change in the PR.
Does that mean the file doesn't exist and this is just an example?

Reply:
Yep, there is no such file anymore.